### DISCUSSION TITLE
Accurately depict special attack regeneration in Colosseum

### DISCUSSION THREAD

**Discussion by AnuVen** [runelite/runelite#17854] [Ideas]
In-game, the special attack is paused in between waves where you are meant to pick an invocation for the following waves, but the special attack orb regeneration timer does not detect it pausing, and appears to go as normal even though it doesn't.

Could we update the orb to accurately depict this?

**Comment by Nightfirecat** [comment]
There isn't any data sent to the client which informs it that special attack regen is paused, so we don't really have a good way to do that. (same for hp regen, etc.)

**Reply by AnuVen** [reply]
is it viable to just pause it upon wave ending / continue upon wave start, or pause upon Minimus (npc that you talk to for invos) spawning and continue timer on despawning
